conda/notices/cache.py

```diff
@@ -81,14 +87,24 @@ def get_notices_cache_dir() -> Path:
 
 
 def get_notices_cache_file() -> Path:
-    """Returns the location of the notices cache file as a Path object"""
+    """
+    Returns the location of the notices cache file as a Path object
```

Comment:
```suggestion
    Return path of notices cache
```
https://peps.python.org/pep-0257/#one-line-docstrings

Type is given as annotation, doesn't need to be repeated in docstring.